### PR TITLE
Add inline svg

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add the helper
- [x] Inline the svg logo instead of using a partial

Based on the work done in #1521

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
```ruby
- ./build/cases/fek-neumunster/index.html
  *  External link https://www.friedrich-ebert-krankenhaus.de/ failed: 503 No error
/home/circleci/website/vendor/bundle/ruby/2.3.0/gems/html-proofer-3.9.1/lib/html-proofer/runner.rb:167:in `print_failed_tests': HTML-Proofer found 1 failure! (RuntimeError)
	from /home/circleci/website/vendor/bundle/ruby/2.3.0/gems/html-proofer-3.9.1/lib/html-proofer/runner.rb:48:in `run'
	from test.rb:17:in `<main>'
Exited with code 1
```

File: helpers/images.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# https://robots.thoughtbot.com/organized-workflow-for-svg
+# https://gist.github.com/bitmanic/0047ef8d7eaec0bf31bb
+# Module to inline svg files with a title
+module Images
+  def inline_svg(filename, options = {})
+    asset = sprockets.find_asset(filename)
+    file = asset.source.force_encoding("UTF-8")
+    doc = Nokogiri::HTML::DocumentFragment.parse file
+    svg = doc.at_css "svg"
+    title = options.delete(:title)
```
Comment (sn3p) [inline helpers/images.rb]: Why delete the title option?

Comment (RonaldDijkstra) [reply helpers/images.rb]: Actually.. no reason

Comment (RonaldDijkstra) [reply helpers/images.rb]: Hmz without it all things break. Will have a look 👍

**Pull request merged**